sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/TestCases/SwitchSectionShouldNotHaveTooManyStatements_CustomValue.cs

```diff
@@ -22,5 +22,21 @@ public Program(int myVariable)
                     break;
             }
         }
+
+        public int Test(string type)
+        {
+            return type switch // Compliant - FN
+            {
+                _ => 1
+                       + 2
```

Comment:
this is not really multiple statements, but a big expression inside - I think we don't have a rule for that ("this statement has too many expressions in it"), and it's out of scope anyway

so in this case, the rule does not really apply to switch expressions (given they cannot have statements inside)